src/python/pants/backend/python/target_types.py

```diff
@@ -291,6 +291,16 @@ def calculate_from_global_options(self, pytest: PyTest) -> Optional[int]:
         return result
 
 
+class PythonTestsForceReruns(BoolField):
+    """Force the tests to rerun, even if they could be satisfied from cache.
+
+    You can also set `--test-force` to temporarily force all tests to rerun.
+    """
+
+    alias = "force_reruns"
```

Comment:
Maybe `force_rerun` or `always_rerun` or `no_cache`? It's not clear what "reruns" (plural) means.

Reply:
I like sticking with `force` for consistency with `--test-force`. I'll use `force_rerun`.